tools/travis/script.sh

```diff
@@ -16,7 +16,13 @@ tools/build_versions.py
 section_end "List.installed.dependencies"
 
 section "Test"
-pytest $TEST_ARGS skimage
+if [[ "$INSTALL_FROM_SDIST" == "1" ]]; then
+    pushd ..
```

Comment:
I don't understand what's happening here.  Can this just be `cmd ; cd ..`?

Reply:
isn't `pushd popd` the same as `cd`??? I can't run pytest from the git directory when it has been installed from source. At least I don't think so.

Reply:
I meant this:

```
(py36) carbo:doc (core-dev-guide*) $ pwd        
/home/stefan/src/scikit-image/doc
(py36) carbo:doc (core-dev-guide*) $ (cd ..; pwd) 
/home/stefan/src/scikit-image
(py36) carbo:doc (core-dev-guide*) $ pwd
/home/stefan/src/scikit-image/doc
```

So that can just be `(cd .. ; pytest ...)`.